crypto/chacha/asm/chacha-x86_64.pl

```diff
@@ -54,9 +54,12 @@
 #
 # Modified from upstream OpenSSL to remove the XOP code.
 
+# The first two arguments should always be the flavour and output file path.
+if ($#ARGV < 1) { die "Not enough arguments provided.
+  Two arguments are necessary: the flavour and the output file path."; }
+
```

Comment:
Nit: We have control of this file and the input arguments. To reduce the merge conflicts from upstream, instead of `die`/`fail the build`, can the `die` code block be replaced with current logics?

Reply:
I explained the reason for this change in all Perl-asm files in a comment above. It significantly simplifies command line parsing in our scripts precisely because we have control of the input arguments. The previous parsing was implemented for cases where we don't have full control and that's why it was complicated and involved even regular expressions matching.